src/loadsave.jl

```diff
@@ -1,37 +1,119 @@
 const sym2loader = Dict{Symbol,Vector{Symbol}}()
 const sym2saver  = Dict{Symbol,Vector{Symbol}}()
-const mimedict   = Dict{Symbol,Vector{Any}}()
+const mimedict   = Dict{Symbol,Vector{@compat(Tuple{DataType, Symbol})}}()
 
-for (appl,fchk,fadd,dct) in (
-        (:applicable_loaders, :check_loader, :add_loader, :sym2loader),
-        (:applicable_savers,  :check_saver,  :add_saver,  :sym2saver))
+function is_installed(pkg::Symbol)
+    isdefined(pkg) && isa(Main.(pkg), Module) && return true # is a module defined in Main scope
+    path = Base.find_in_path(string(pkg)) # hacky way to determine if a Package is installed
```

Comment:
Sadly not... It takes 12s on windows and allocates 8mb... Also it errors for mysterious reasons... I suppose it errors when it can't find it in METADATA;)